README.md

```diff
@@ -1 +1,15 @@
-# igniteui-angular-i18n
\ No newline at end of file
+# igniteui-angular-i18n
+
+[![Build Status](https://travis-ci.org/IgniteUI/igniteui-angular-i18n.svg?branch=master)](https://travis-ci.org/IgniteUI/igniteui-angular-i18n)
+
+## Purpose
+
+This respository stores the Japanese and Korean resource strings for the Ignite UI for Angular `i18n`. The English resource strings are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository. It also contains the interfaces for each localizable component's `resourceStrings` property. They can be used as a schema for creating custom resource strings.
+
+## File Usage
+
+An npm package will be produced and it will be referenced in the Ignite UI for Angular product.
+
+## Branches
+
+The branching scheme must follow the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) branching.  Work in the master branch, and when we are getting ready for a release, create a versioned branch (ex. 6.2.x). This will allow the build to get the correct version of the files.
```

Comment:
Instead of this:
`Work in the master branch, and when we are getting ready for a release, create a versioned branch (ex. 6.2.x). This will allow the build to get the correct version of the files.`
you could mention that in order to get the correct resource the `igniteui-angular` and `igniteui-angular-i18n` packages should have the same version in the clients' projects.